src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1208,7 +1208,7 @@ void BlockchainLMDB::open(const std::string& filename, const int db_flags)
   if (is_hdd_result)
   {
     if (is_hdd_result.value())
-        MCLOG_RED(el::Level::Warning, "global", "The blockchain is on a rotating drive: this will be very slow, use a SSD if possible");
+        MCLOG_RED(el::Level::Warning, "global", "The blockchain db file might be on a rotating disk(HDD): this will be very slow, use a SSD if possible");
```

Comment:
https://github.com/monero-project/monero/pull/4639#discussion_r226080953
```suggestion
        MCLOG_RED(el::Level::Warning, "global", "The blockchain db file might be on a rotating disk(HDD): this will be very slow, use an SSD if possible");
```